convert-hf-to-gguf.py

```diff
@@ -324,7 +324,7 @@ def _set_vocab_sentencepiece(self):
 
         if not tokenizer_path.is_file():
             print(f'Error: Missing {tokenizer_path}', file=sys.stderr)
```

Comment:
Actually, this line should probably deleted, since it shouldn't be printed when the exception is caught, and when it isn't, the exception is sufficient:
```suggestion
```

Reply:
Right, that line is unnecessary.